server/dataStruct/struct.go

```diff
@@ -1,12 +1,15 @@
 package datastruct
 
 type SystemInfo struct {
```

Comment:
this struct is being duplicated across client and server code. 
client should import it from a server package for use.